openedx/core/djangoapps/course_roles/models.py

```diff
@@ -0,0 +1,104 @@
+"""
+Models for course roles schema
+"""
+from django.contrib.auth import get_user_model
+from django.db import models
+
+from openedx.core.djangoapps.content.course_overviews.models import CourseOverview
+from organizations.models import Organization
+
+User = get_user_model()
```

Comment:
In general, for the purposes of `models.py` Django docs recommend that you use `settings.AUTH_USER_MODEL` instead of `get_user_model()`: https://docs.djangoproject.com/en/3.2/topics/auth/customizing/#django.contrib.auth.get_user_model